src/swtpm/swtpm_nvfile.c

```diff
@@ -606,8 +632,13 @@ static TPM_RESULT SWTPM_NVRAM_GetFilenameForName(char *filename,        /* outpu
         break;
     }
 
-    n = snprintf(filename, bufsize, "%s/tpm%s-%02lx.%s",
-                 state_directory, suffix, (unsigned long)tpm_number, name);
+    if (is_tempfile) {
+        n = snprintf(filename, bufsize, "%s/TPM%s-%02lx.%s",
```

Comment:
I hope that nobody ever gets an idea to run swtpm on a case-insensitive file system...

Reply:
Will change this to `TMP` :-)